.changeset/many-singers-complain.md

```diff
@@ -0,0 +1,7 @@
+---
+'@lit/localize-tools': minor
+---
+
+**BREAKING** Update analysis to consider messages with same id **and** description to be identical (but no longer checks for expressions to be same) and improve error message on finding incompatible duplicates.
+
+`lit-localize extract` will now error if multiple messages had the same text but different `desc` option. Be sure to add the same `desc` option for these messages to be considered the same translatable message or add different `id` options to differentiate them.
```

Comment:
```suggestion
`lit-localize extract` will now error if multiple messages had the same text but different `desc` option. Be sure to add the same `desc` option for these messages to be considered the same translatable message or add different `id` options to differentiate them.

**BREAKING** XLB files now require `name` attributes on all `<ph>` elements. If you encounter errors relating to missing `name` attributes, run `lit-localize extract` to re-generate your XLB files from source with new `name `attributes.
```

Reply:
Oops sorry, I must have just scrolled past that by mistake. Nope, what you already have is great!